### DISCUSSION TITLE
How to Find Record by Id?

### DISCUSSION THREAD

**Discussion by tailorck** [Budibase/budibase#5338] [Help] [answered]
Hey Budibase team,

I am the CEO & Co-Founder of an education technology company. We are very excited to use Budibase for rapid development and to expose our product to more universities. 

I am creating a simple dashboard application right now to allow professors to view the information we have collected on their courses. The home screen uses a Card Block to display all the courses and by clicking on the Title Link, they can navigate to the Course Dashboard (url = /dashboard/:id).

I created a Breadcrumb on the /dashboard screen and would like to look up the name of the Course inside my Course table by the id which was passed in the URL from the home screen. What is the preferred way of going about doing this?

**Comment by melohagan** [comment, marked as answer]
Hey @tailorck 

Thanks for your interest in Budibase!

It may be possible to send the course name in the URL alongside the id. 
Similar to what was done with the service name here: https://github.com/Budibase/budibase/discussions/5299#discussioncomment-2530261
